mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml

```diff
@@ -333,5 +333,12 @@
         <Required>N</Required>
       </exceptions>
     </surbl>
+    
+    <multimap>
+      <badfileextension type="CSVListField">
+        <Required>Y</Required>
```

Comment:
Is Required Y correct when someone dont want to use this feature?

Reply:
good point will make it a N